apps/erp/app/styles/nprogress.css

```
@@ -4,13 +4,40 @@
+     loader stalls at one percentage. Tiles because NProgress translates .bar. */
+  background-image: linear-gradient(
+    90deg,
+    transparent 0%,
+    #7f9cffe6 50%,
+    transparent 100%
+  );
+  background-size: 200px 100%;
+  background-repeat: repeat-x;
+  animation: nprogress-shine 0.9s linear infinite;
 
   position: fixed;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)